barretenberg/cpp/src/barretenberg/dsl/acir_format/honk_recursion_constraint.hpp

```diff
@@ -20,7 +19,7 @@ using namespace bb::plonk;
  *
  * @param verification_key_data The inner circuit vkey. Is converted into circuit witness values (internal to the
  * backend)
- * @param proof The plonk proof. Is converted into circuit witness values (internal to the backend)
+ * @param proof The honk proof. Is converted into circuit witness values (internal to the backend)
```

Comment:
Probably could update these comments in general as the nested agg object is not part of the `HonkRecursionConstraint`. But this in non-blocking and can come in a follow-up